var/spack/repos/builtin/packages/rocprofiler-systems/package.py

```diff
@@ -0,0 +1,169 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class RocprofilerSystems(CMakePackage):
+    """Application Profiling, Tracing, and Analysis"""
+
+    homepage = "https://github.com/ROCm/rocprofiler-systems"
+    git = "https://github.com/ROCm/rocprofiler-systems.git"
+    url = "https://github.com/ROCm/rocprofiler-systems/archive/refs/tags/rocm-6.3.1.tar.gz"
+
+    maintainers("dgaliffiAMD", "afzpatel", "srekolam", "renjithravindrankannath", "jrmadsen")
+
+    license("MIT")
+
+    version("amd-mainline", branch="amd-mainline", submodules=True)
+    version("amd-staging", branch="amd-staging", submodules=True)
+    version(
+        "rocm-6.3.1",
```

Comment:
The Spack version strings also need to be changed (but don't change the `tag=` part)

```suggestion
        "6.3.1"
```